src/i18n/locale/en.json

```diff
@@ -199,17 +199,17 @@
     },
     "date": {
       "label": {
-        "today": "TODAY",
+        "today": "Today",
         "today_at": "Today at {{timeString}}",
-        "yesterday": "YESTERDAY",
+        "yesterday": "Yesterday",
         "yesterday_at": "Yesterday at {{timeString}}"
       }
     },
     "error": {
       "label": {
-        "invalid_action": "This is not a valid Action",
-        "invalid_file_type": "invalid file type!",
-        "update_failed": "Checking for updates failed!"
+        "invalid_action": "This is not a valid action",
+        "invalid_file_type": "invalid filetype",
```

Comment:
I don't think file type is written as one word

Reply:
In English both are OK, but typically not in other languages.
https://en.wiktionary.org/wiki/filetype
Helps avoid errors in translation.